server/controllers/traineeController.js

```diff
@@ -1,48 +1,63 @@
 import db from "../db";
 
 export const getAllTrainees = (req, res) => {
-    res.status(200).json("hello from trainees route");
+	res.status(200).json("hello from trainees route");
 };
 
 export const getTrainee = async (req, res) => {
-    const { traineeId } = req.params;
```

Comment:
we don't have traineeId in the table, its only id

Reply:
@Zobeir-Rigi yes you are right, we discussed it with Boshra, I should change it.

Reply:
@Zobeir-Rigi yes you are right, we discussed it with Boshra, I should change it.

Reply:
@OlenaReukova  yes you change it to id , just one point when you equal something with req.params its not related to data base , this is something regarding your endpoint and request.